### PR TITLE
Add/Remove Book

### PR DESCRIPTION
**Pull request by kensayo**
- Add books using Redux
- Remove books using Redux
- Implementation of reducers

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello  

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/bookList.js [hunk 1]
```diff
@@ -1,30 +1,38 @@
 import React from 'react';
+// import { useSelector } from 'react-redux';
```
Comment (mikenath223) [inline src/components/bookList.js]: - [ ] Please remove this code comment.

File: src/components/bookList.js [hunk 2]
```diff
@@ -1,30 +1,38 @@
...
-      <h1>Lorem Ipsun</h1>
-      <span>Subtittle</span>
-      <div>
-        <ul>
-          <li>Comments</li>
-          <li>Remove</li>
-          <li>Edit</li>
-        </ul>
+const BookList = () => {
+  // const bookStore = useSelector((state) => state.book);
```
Comment (mikenath223) [inline src/components/bookList.js]: - [ ] Please remove this code comment.

**Comment by mikenath223** [comment]
- [ ] Please update the readme following [this template](https://github.com/microverseinc/readme-template).

File: src/components/addBook.js [hunk 3]
```diff
@@ -1,18 +1,35 @@
...
+
+  return (
+    <div className="mx-auto m-4">
+      <h2>ADD NEW BOOK</h2>
+      <div>
+        <input type="text" id="title" placeholder="Book title" />
+        <select type="select" id="category">
+          <option value="Lorem 1">Lorem 1</option>
+          <option value="Lorem 2">Lorem 2</option>
+          <option value="Lorem 3">Lorem 3</option>
```
Comment (mikenath223) [inline src/components/addBook.js:27]: - [ ] The functionality to **Add** and **Remove** a book doesn't work. Please check and fix it.

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @kensayo, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**